app/assets/stylesheets/marking_schemes.scss

```diff
@@ -0,0 +1,17 @@
+@import 'common/core';
+
+table {
```

Comment:
I was wondering why he didn't use a common table style... @arkon do we not have one?